src/Entity/User.php

```diff
@@ -41,21 +42,25 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface, \Serial
     #[ORM\Column(type: Types::STRING)]
     private ?string $password = null;
 
+    #[Assert\When(
+        expression: 'this.getPlainPassword() not in [null, ""]',
+        constraints: [
+            new UserPassword(message: 'user.current_password.invalid'),
+        ],
+        groups: ['UserEdit'],
+    )]
+    private ?string $currentPassword = null;
+
     #[Assert\Regex(
         pattern: '/^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$/',
         message: 'user.password.valid'
     )]
     #[Assert\NotCompromisedPassword(message: 'user.password.not_compromised')]
-    #[Assert\When(
-        expression: 'this.getPassword() in [null, ""]',
-        constraints: [
-            new Assert\NotBlank(message: 'user.password.not_blank'),
-        ],
-    )]
+    #[Assert\NotBlank(message: 'user.password.not_blank', groups: ['Registration'])]
     private ?string $plainPassword = null;
 
     #[ORM\Column(type: Types::BOOLEAN)]
-    private bool $isVerified = false;
+    private bool $verified = false;
```

Comment:
Tu utilises pas le prefix "is" pour un booléen ?

Reply:
Si mais pour le getter, là on avait `isVerified` en property et en getter du coup c'était bizarre